.github/workflows/compatibility-report.yml

```diff
@@ -54,17 +54,59 @@ jobs:
           path: compatibility.md
 
       # Generate the new PR to update the doc in the repo.
-      - run: |
-          git config user.name $GITHUB_ACTOR
-          git config user.email gh-actions-${GITHUB_ACTOR}@github.com
       - run: |
           rm -f status.log
       - uses: peter-evans/create-pull-request@6d6857d36972b65feb161a90e484f2984215f83e
         with:
+          token: ${{ secrets.NODE_AGENT_GH_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
not sure i following why you're checking an or here. shouldn't we always use our bot token which is `NODE_AGENT_GH_TOKEN`?

Reply:
As a guard for forks. I'm really not sure what happens if it is run from a fork that doesn't have our secrets.